lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -552,6 +552,12 @@ module.exports = class MethodMappings {
           return this.driver.manage().getCookies();
         },
 
+        async getCookieString(name) {
```

Comment:
this should be called simply `getCookie` since we're not returning the whole cookie string anymore.

Reply:
Yup, changed to `getCookie`